packages/mds-db/events.ts

```diff
@@ -483,3 +493,128 @@ export async function getMostRecentEventByProvider(): Promise<{ provider_id: UUI
   const sql = `select provider_id, max(recorded) from ${schema.TABLE.events} group by provider_id`
   return makeReadOnlyQuery(sql)
 }
+
+// for CMM
+type TimeRange = {
+  start: Timestamp
+  end: Timestamp
+}
+interface GetVehicleEventsFilterParams {
+  vehicle_types?: VEHICLE_TYPE[]
```

Comment:
You really like plural for these? The values are intended to be an OR so I think singular makes sense because each of these is singular on the event (except `geography_id`). I could definitely be swayed though, especially by any MDS precedent.

 I guess there's a valid english reading of either singular or plural (e.g., "I'm searching for events whose `propulsion_type` is either `electric` or `human`" vs "I'm searching for events with propulsion types `electric` or `human`"). cc @levi217 @cjlynch12 @EthanSherr for opinions.

Reply:
I'm fine with either, as long as it's consistent.